packages/datadog-plugin-vm/test/integration-test/client.spec.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+const {
+  useSandbox, sandboxCwd, varySandbox, curl,
+  FakeAgent, spawnPluginIntegrationTestProc
+} = require('../../../../integration-tests/helpers')
+const { assert } = require('chai')
```

Comment:
Could you please change these to use Node.js assert instead in all of these files?

```suggestion
const assert = require('node:assert/strict')
```